jenkins/values.yaml

```diff
@@ -0,0 +1,219 @@
+# Default values for jenkins.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+## Overrides for generated resource names
+# See templates/_helpers.tpl
+# nameOverride:
+# fullnameOverride:
+
+Nginx:
+  Image: "nginx"
+  ImageTag: "stable-alpine"
+  ImagePullPolicy: "Always"
+  Port: 9090
+  ServiceType: LoadBalancer
```

Comment:
Maybe better to use `NodePort` here. Type `LoadBalancer` assumes it runs in a cloud with a loadbalancer service

Reply:
We can specify that in our values on deployment (on the terraform module). That way we minimize deviations from upstream.